src/UsgsAstroLsSensorModel.cpp

```diff
@@ -2664,57 +2660,56 @@ std::string UsgsAstroLsSensorModel::constructStateFromIsd(const std::string imag
 
    int num_params = NUM_PARAMETERS;
 
-   state["m_imageIdentifier"] = isd.at("IMAGE_ID");
-   state["m_sensorType"] = isd.at("SENSOR_TYPE");
-   state["m_totalLines"] = isd.at("TOTAL_LINES");
-   state["m_totalSamples"] = isd.at("TOTAL_SAMPLES");
+   state["m_imageIdentifier"] = "UNKNOWN";
+   state["m_sensorType"] = "LINE_SCAN";
+   state["m_totalLines"] = isd.at("image_lines");
+   state["m_totalSamples"] = isd.at("image_samples");
    state["m_offsetLines"] = 0.0;
    state["m_offsetSamples"] = 0.0;
-   state["m_platformFlag"] = isd.at("PLATFORM");
-   state["m_aberrFlag"] = isd.at("ABERR");
-   state["m_atmRefFlag"] = isd.at("ATMREF");
+   state["m_platformFlag"] = 1;
+   state["m_aberrFlag"] = 0;
+   state["m_atmRefFlag"] = 0;
+   state["m_centerEphemerisTime"] = isd.at("center_ephemeris_time");
 
-   state["m_startingEphemerisTime"] = isd.at("STARTING_EPHEMERIS_TIME");
-   state["m_centerEphemerisTime"] = isd.at("CENTER_EPHEMERIS_TIME");
-
-   if (isd.find("NUMBER_OF_INT_TIMES") == isd.end()) {
+   if (isd.find("number_of_int_times") == isd.end()) {
      state["m_intTimeLines"] = {0.5};
-     state["m_intTimeStartTimes"] = {state["m_startingEphemerisTime"].get<double>() - state["m_centerEphemerisTime"].get<double>()};
-     state["m_intTimes"] = {isd.at("INT_TIME").get<double>()};
+     state["m_intTimes"] = {isd.at("int_time").get<double>()};
+     state["m_intTimeStartTimes"] = {state["m_totalLines"].get<double>() * state["m_intTimes"][0].get<double>() / 2 };
    }
    else {
-     int numIntTimes = isd.at("NUMBER_OF_INT_TIMES");
+     int numIntTimes = isd.at("number_of_int_times");
      for (int i = 0; i < numIntTimes; i++) {
-       state["m_intTimeLines"].push_back(isd.at("INT_TIME").at(i*3));
-       state["m_intTimeStartTimes"].push_back(isd.at("INT_TIME").at(i*3+1));
-       state["m_intTimes"].push_back(isd.at("INT_TIME").at(i*3+2));
+       // TODO what does this look like under the new spec?
+       state["m_intTimeLines"].push_back(isd.at("int_time").at(i*3));
+       state["m_intTimeStartTimes"].push_back(isd.at("int_time").at(i*3+1));
+       state["m_intTimes"].push_back(isd.at("int_time").at(i*3+2));

```

Comment:
@Kelvinrr should weigh in here on the specification: https://github.com/USGS-Astrogeology/pfeffernusse/blob/master/swagger.yaml

I *think* this is all inside of the `line_scan_rate` entry in the spec? This comment is for lines 2676 through 2685.